src/app/dashboard-widgets/add-codebase-widget/add-codebase-widget.component.html

```diff
@@ -3,14 +3,18 @@
     <div class="card-pf-heading f8-card-heading">
       <f8-feature-toggle featureName="AppLauncher">
         <div class="card-pf-heading-details f8-card-heading-details" default-level>
-          <a id="spacehome-codebases-add-space-button" class="btn btn-link f8-card-heading-btn-link" (click)="addToSpace.emit()">
-            <i class="pficon pficon-add-circle-o"></i>
-          </a>
-        </div>
-        <div class="card-pf-heading-details f8-card-heading-details" user-level>
+          <ng-container *ngIf="userOwnsSpace">
             <a class="btn btn-link f8-card-heading-btn-link" (click)="addToSpace.emit()">
-                <i class="pficon pficon-add-circle-o"></i>
+              <i id="default-level-add-to-space-btn" class="pficon pficon-add-circle-o"></i>
             </a>
+          </ng-container>
+          <div class="card-pf-heading-details f8-card-heading-details" user-level>
+            <ng-container *ngIf="userOwnsSpace">
+              <a class="btn btn-link f8-card-heading-btn-link" (click)="addToSpace.emit()">
+                <i id="user-level-add-to-space-btn" class="pficon pficon-add-circle-o"></i>
+              </a>
+            </ng-container>
```

Comment:
For a component like work-item-widget.component, ng-container looks like a good fit because it's used in just one place. However, add-codebase-widget.component uses several ng-container tags throughout the HTML layout.

To complicate things, the HTML layout is already wrapped in multiple places for the AppLauncher feature toggle. That forces us to wrap each user-level and default-level directive with multiple ng-container tags.

Considering most of the functionality in this component is being wrapped with ng-container, could we create an alternate, simplified layout for this scenario? I believe that would help make the code more readable and maintainable.

Alternatively, we could remove the AppLuncher feature -- looks like that's not used anymore?